bin/cli

```diff
@@ -225,8 +225,13 @@ function warnIfNpmStart (argv, env) {
 
       const projectDirectory = process.argv[4]
 
+      // extract the name from the original package.json
+      const pkgJson = await fs.readJson(path.join(projectDirectory, 'package.json'))
+      // if the name doesn't exist, use the name of the project directory
+      const { name = projectDirectory.split('/').pop() } = pkgJson || {}
```

Comment:
Keeping the existing name seems like a good idea, however I'm not so sure about creating a name if one doesn't exist; I'd prefer to keep it undefined.

Reply:
Done